.github/workflows/publish-linux-packages.yml

```diff
@@ -0,0 +1,118 @@
+name: Publish Linux Packages
+
+on:
+  workflow_call:
+    inputs:
+      ziti-version:
+        description: semver string generated by 'ziti-ci get-current-version'
+        type: string
+        required: true
+
+jobs:
+  publish-linux-packages:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        package_name:
+          - ziti-cli
+        arch:
+          - goreleaser: amd64
+            gox: amd64
+            deb: amd64
+            rpm: x86_64
+          - goreleaser: arm64
+            gox: arm64
+            deb: arm64
+            rpm: aarch64
+          - goreleaser: armv7
+            gox: arm
+            deb: armv7
+            rpm: armv7
+        nfpm_packager:
+          - rpm
+          - deb
+          # - archlinux
+          # - apk
+    env:
+      ZITI_VERSION: ${{ inputs.ziti-version || github.event.inputs.ziti-version }}
+      ZITI_MAINTAINER: "OpenZiti Maintainers <developers@openziti.org>"
+      ZITI_HOMEPAGE: "https://openziti.io"
+      ZITI_VENDOR: "NetFoundry"
+      GOARCH: ${{ matrix.arch.goreleaser }}
+    steps:
+      - name: Checkout Workspace
+        uses: actions/checkout@v3
+
+      - name: Download Linux Release Artifacts
+        uses: actions/download-artifact@v3
+        with:
+          name: linux-release-${{ github.run_id }}
+          path: release/
+
+      - run: mv -v ./release/${{ matrix.arch.gox }}/linux/ziti ./release/ziti
+
+      - name: Build Package
+        id: nfpm
+        # uses: burningalchemist/action-gh-nfpm@v1
```

Comment:
```suggestion
        uses: burningalchemist/action-gh-nfpm@v1
```

Reply:
They accepted and released the patch that motivated me to fork the nfpm action so we can revert back to the upstream